repo_guidelines.md

```diff
@@ -132,6 +136,60 @@ See the [`LICENSE`](/LICENSE) file for details.
  */
 ```
 
+### Core Issue Label Configuration
+EDGI repos use a minimum/core set of standard issue labels.
+
+Labels are applied using the NPM package [github-labels](https://www.npmjs.com/package/github-labels) using the following `configuration.json`:
```

Comment:
Would it be useful to link to Brendan's [gist](https://gist.github.com/b5/d3bd0de5aa88842b2455a2e5c1112c87)?
Particularly the script with notes

Reply:
I was hoping to get it working with just the standard documentation, but it hasn't been going well for me with that or with the gist! Digging into it a bit more.